css/bs/bootstrap-grid.css

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Can you give these files an extension of `.scss` instead and delete the error comments?